gluten-ut/spark34/src/test/scala/org/apache/gluten/utils/velox/VeloxTestSettings.scala

```diff
@@ -717,6 +718,7 @@ class VeloxTestSettings extends BackendTestSettings {
     .exclude(
       "SPARK-26677: negated null-safe equality comparison should not filter matched row groups")
   enableSuite[GlutenParquetV2QuerySuite]
+    .exclude("row group skipping doesn't overflow when reading into larger type")
```

Comment:
Do you know why this test fails? Please file one issue for it if necessary.

Reply:
Gluten with latest Spark 3.3.x and 3.5.x all disable it, I will open an issue, but I dont know the reason.

Reply:
#7128